packages/manager/modules/vps/src/dashboard/additional-disk/upgrade/upgrade.routing.js

```diff
@@ -0,0 +1,17 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('vps.detail.dashboard.additional-disk-upgrade', {
+    url: '/additional-disk-upgrade',
+    views: {
+      modal: {
+        component: 'ovhManagerVpsComponentAdditionalDiskUpgrade',
+      },
+    },
+    layout: 'modal',
+    resolve: {
+      upgradableDisks: /* @ngInject */ (catalog, vpsLinkedDisk, VpsService) =>
+        VpsService.getUpgradableAdditionalDisk(catalog, vpsLinkedDisk),
+
+      goBack: /* @ngInject */ ($state) => () => $state.go('^'),
```

Comment:
Question: not showing success or error message?

Reply:
Yep i display them via `CucCloudMessage` module.

Reply:
No, not rquired, it managed by the `cucCloudMessage` module.